UKHomeOffice/sgmr-service
# NSMW-34 Add autocomplete field to pleasure craft form

## AC

Replace the existing autocomplete component with the AlphaGov version that we use on nmsw

----

## To test

- go to pleasure craft page
- add a new craft
- select yes to country of registration
- start typing in field
- > country options should be shown
- select a country
- refresh the page
- > country option should persist
- save craft
- click update for that craft
- > country should be selected to yes
- > country information should be shown
- click change country
- > input should be shown
- start typing in field
- > country options should be shown
- select a country
- refresh the page
- > country option should persist
- save craft
- click update for that craft
- > changes should have been saved
- change country to no
- save craft
- click update for that craft
- > country should be selected to no
- click yes
- > input field should be shown

----

## Notes
